content/en/blog/basics-for-passwordless/index.md

```diff
@@ -0,0 +1,136 @@
+---
+date: 2024-01-05
+title: "Path to Passwordless"
+linkTitle: "Path to Passwordless"
+description: "Passwordless user authentication is the new black. After building decades of password-based authentication flows, web application developers must familiarize themselves with this new technology. Luckily, there is already a variety of proprietary and open-source tooling one can choose from."
+author: Laura Vuorenoja
+resources:
+  - src: "**.{png,jpg}"
+    title: "Image #:counter"
+---
+
+Passkeys and security keys have gained more and more popularity lately, and no wonder – they provide
+much more security and usability than passwords. These authentication methods utilize strong and
+phishing-resistant public key credentials that the keys or authenticators, as we call them,
+can create automatically. However, implementing passwordless support in your web application,
+or even replacing passwords altogether, might initially seem overwhelming.
```

Comment:
"Overwhelming" term refers to the fact that if the passwordless technology is unfamiliar, and you have implemented an existing application with traditional username-password-based authentication, it might seem quite a big job to undertake implementing new routines for the credential handling, both in front-end and back-end side.